.github/workflows/release-actions.yml

```diff
@@ -0,0 +1,35 @@
+name: Release actions
+
+on:
+  release:
+    types: released
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Tag review replyers in closed issues with 'Needs: Review reply' label"
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const issue_number = context.issue.number;
+            const { data: issues } = await github.rest.issues.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'closed',
+              labels: ['Needs: Review reply']
+            });
+
+            if (issues.length > 0) {
+              for (const issue of issues) {
+                await github.rest.issues.createComment({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: issue.number,
+                  body: 'This feature or fix is now released! @AntennaPod/review-replyers, time to inform relevant reviews in Google Play console (please see links in one or more of the comments above).'
```

Comment:
Hmm, I think this part is quite hard to do. We often close issues on the develop branch after the feature freeze of the beta version. So this would tag things that are not actually released.

Reply:
Thanks for taking a look.
This workflow only runs when you create a public release, i.e. when it's available on Google Play. Only at that point it posts messages on closed issues (thus not as soon as the issue gets closed).

Reply:
Yup but when creating the release, we have already closed additional issues that are not in that release but the release after that

Reply:
Ah, that's what you mean. Right.

Then either we should start linking issues with milestones, or I get tagged twice on some issues (which is not a huge problem).